CHANGES.md

```diff
@@ -25,6 +25,8 @@
 * Compiler: drop support for OCaml 4.03 and bellow
 * Lib: add offsetX and offsetY to Dom_html.mouseEvent
 * Lib: add innerText property for Dom_html
+* Lib: add messageEvent to Dom_html
+* Lib: add PerformanceObserver API
```

Comment:
The changelog entry is misplaced

Reply:
I moved the changelog entries to the section at the top. I thought I had corrected that mistake when I re-created the PR, but I guess not. I'm a little unsure that I've put them in the correct place still, because they're under the section dated 2021-10-06, which makes me think the release already happened.

Reply:
If you rebase your PR, you'll see that there is a new section for unrelated changes

Reply:
Ah, I see. I've rebased (without introducing a merge commit this time) and moved the changelog entries to the right place.